hacks/cutNewVersions.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash   
+
+# Before running this script make sure these versions are correct!
+NEW_DOCS_VERSION="8.1"
+PREVIOUS_DOCS_VERSION="8.0"
+NEW_OPTIMIZE_VERSION="3.9.0"
+PREVIOUS_OPTIMIZE_VERSION="3.8.0"
```

Comment:
What happens if these are not modified and the script runs twice with the same versions?

Reply:
Fun question!

It looks like nothing bad happens -- the `docusaurus version` calls both fail with an error; the `sed` calls all succeed, but they wouldn't have found any matches for their regexes, because they'd been replaced already. 

<img width="797" alt="image" src="https://user-images.githubusercontent.com/1627089/193360862-a6220f72-40cb-4eba-8748-8f455eea2bd0.png">

But that feels lucky. I think I'll add a flag to the top of the script to exit upon any error -- in case something changes in the future that the script isn't completely harmless when re-run.

(That screenshot also reminded me that I wanted to add the number of total steps in the echo statements, so I'll add those also.)

Reply:
> But that feels lucky. I think I'll add a flag to the top of the script to exit upon any error -- in case something changes in the future that the script isn't completely harmless when re-run.

Love this! Thank you!